src/features/TenantSupport/TenantSupport.tsx

```diff
@@ -24,6 +24,8 @@ import {
 const issues = [
   {
     category: 'Noise Complaint',
+    backendCategory: 'noise',
+
```

Comment:
Question: wonder how this extra line ended up here? Should have been caught w linting